### PR TITLE
DB-10373 sqlshell buildin commands adhere to maximumdisplaywidth

### PR DESCRIPTION
**Pull request by martinrupp**
check commit 1 for functional change,
commit 2 is refactoring

### CODE REVIEW

**Comment by martinrupp** [comment]
please check commit 1 for functional change, commit 2 is refactoring

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Comment by spliceci** [comment]
TEST FAILED
http://nexus.splicemachine.com:8080/job/spliceengine-PR/job/Splice-PR-Branch/1633/

**Comment by martinrupp** [comment]
jenkins please test branch @cdh6.3.0,skipTestsLongerThan2Minutes

**Review by dgomezferro — APPROVED** [review]
This is great, thanks @martinrupp !

File: db-tools-ij/src/main/grammar/ij.jj [hunk 1]
```diff
@@ -860,19 +814,13 @@ class ij {
...
 
             // Small subset of the result set fields available
-            int[] displayColumns = new int[] {
-                rs.findColumn("COLUMN_NAME"),
-                rs.findColumn("TYPE_NAME"),
-                rs.findColumn("ORDINAL_POSITION")
+            int[] columnParameters = new int[] {
+                rs.findColumn("COLUMN_NAME"), 32, 0,
+                rs.findColumn("TYPE_NAME"), 16, 32,
+                rs.findColumn("ORDINAL_POSITION"), 16
```
Comment (ascend1) [inline db-tools-ij/src/main/grammar/ij.jj]: Is there one more number missing after 16? I didn't try it but it feels like index arithmetic like `i*3+2` later would be messed up.

Comment (martinrupp) [reply db-tools-ij/src/main/grammar/ij.jj]: good catch!

**Comment by spliceci** [comment]
TEST SUCCEEDED +1

File: db-tools-ij/src/main/grammar/ij.jj (hunk 1 above)
Comment (martinrupp) [reply db-tools-ij/src/main/grammar/ij.jj]: i thought i can do this in a "simple" way but that was too simple. i fixed this now with a builder-pattern

**Review by ascend1 — APPROVED** [review]

**Review by carolp-503 — APPROVED** [review]
+1

**Pull request merged**